src/category_theory/functor.lean

```diff
@@ -2,18 +2,18 @@
 Copyright (c) 2017 Scott Morrison. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Tim Baumann, Stephen Morgan, Scott Morrison
+-/
+import tactic.reassoc_axiom
+import tactic.monotonicity
 
-Defines a functor between categories.
+/-!
+# Functors
 
-(As it is a 'bundled' object rather than the `is_functorial` typeclass parametrised
-by the underlying function on objects, the name is capitalised.)
+Defines a functor between categories, extending a `prefunctor` between quivers.
 
-Introduces notations
-  `C ⥤ D` for the type of all functors from `C` to `D`.
-    (I would like a better arrow here, unfortunately ⇒ (`\functor`) is taken by core.)
+Introduces notation `C ⥤ D` for the type of all functors from `C` to `D`.
+(I would like a better arrow here, unfortunately ⇒ (`\functor`) is taken by core.)
```

Comment:
First person here is pretty weird. I assume nobody will git blame that to figure out the subject.

Reply:
Haha, this probably predates the existence of mathlib. :-) I'll update.

Reply:
```suggestion
(Unfortunately the ⇒ arrow (`\functor`) is taken by core, but in mathlib4 we should switch to this.)
```